### ISSUE TITLE
Cannot access tab interface with keyboard.mp4

### ISSUE THREAD

**Issue by jill-iterators** [CityOfBoston/Iterators#141]
Screengrab/videos (insert file names here)
2020 08 24, Cannot access tab interface with keyboard.mp4

Link
https://d8-stg.boston.gov/departments/emergency-medical-services/cpr-training#group-cpr-certifications

Node # (if known)

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > Mac 10.15.6 > Chrome 84

Trusted Tester Test
n/a

Brief Description
While trying to access the tab interface with the keyboard only, tester was unable to land on the tabs.

Steps to reproduce
1)	00:18, Tab onto the City of Boston link
2)	00:19, Tab onto Mayor Walsh link, then Pay and Apply, Public Notices, Feedback, Translate and Search.
3)	00:24, Continue to tab successfully to the Coronavirus updates and Department Information.
4)	00:27, Click on the tab button again and land on “Learn About Hands Only CPR”, which skips to another page
a)	Note: This is an accessibility bug as tab should have moved to three-tab interfaces labeled Community Hands Only CPR, then Hands Only CPR Video, then Group CPR Certifications.

Desired Result
Able to tab through the page using the tab key only and successfully lands on all items successfully.

Actual Result
Unable to land on the three-tabs on the page using the tab key only.

[2020 08 24, Cannot access tab interface with keyboard trimmed.mp4.zip]

**Comment by subaha-cob** [comment]
Test on: https://d8-uat.boston.gov/

**Comment by jill-iterators** [comment]
This is working as expected now.
